versions/v6/static/scss/templates/_story-default.scss

```diff
@@ -2,98 +2,34 @@
 // Stories - Default template
 //
 
-.story-default {
+.story-template-default {
```

Comment:
Consider making this selector more specific (e.g. `.story-template-default .story { ....`) to prevent these font/line height adjustments from bleeding into the site header/footer and "More in this Issue" subfooter; or add a unique class for the story lead/deck and scope all style overrides to it, `.story-content` and `.story-sidebar` instead.

Reply:
Thanks! I didn't realize these font changes were bleeding into the other parts of the page 😅 I already had the font-family adjustment scoped to the lead, story-content, and byline so I've moved those there and that looks better.